.github/workflows/create-token-pr.yaml

```diff
@@ -0,0 +1,58 @@
+###
+#
+# Creates a PR from the token branch whenever is has commits ahead of main.
+#
+###
+
+name: Create Token PR
+on:
+  push:
+    branches:
+      - tokens/v*
+
+jobs:
+  create_pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checkout the token branch which was pushed
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      # Get the name of the token branch which was pushed and the corresponding PR branch
+      - name: Get Branch
+        id: branch-names
+        run: |
+          echo "token-branch=tokens/$(echo ${GITHUB_REF##*/})"    >> $GITHUB_OUTPUT
+          echo "pr-branch=merge-tokens-$(echo ${GITHUB_REF##*/})" >> $GITHUB_OUTPUT
+
+      # Check if the PR branch already exists
+      - name: Check PR Branch
+        id: git-branch
+        run: |
+          GIT_BRANCH="$(git branch -l $PR_BRANCH)"
+          echo "pr_branch=${GIT_BRANCH}" >> $GITHUB_OUTPUT
+        env:
+          PR_BRANCH: ${{ steps.branch-names.outputs.pr-branch }}
+
+      # if the branch already exits, update the PR
+      - name: Update Branch
+        if: steps.git-branch.outputs.pr_branch != ''
+        run: |
+          git checkout $PR_BRANCH
+          git merge $TOKEN_BRANCH -X theirs --no-edit
+          git push
```

Comment:
Would be nice, if the `Swiss Post Bot` could be used as the author. See `release.yaml` -> `Commit Changes and Push Branch` how this can be done.

Reply:
I don't think I can use EndBug/add-and-commit@v9 here since I am not creating a new commit but a merge commit.

Reply:
I added the GITHUB_TOKEN which should do the same.